src/Router.cpp

```diff
@@ -5,15 +5,35 @@ Router::Router()
 {
 }
 
+Router::Router(ServerBlock serverBlock) : _serverBlock(serverBlock)
+{
+}
+
+Router::Router(const Router &obj)
+{
+	*this = obj;
+}
+
+Router &Router::operator=(const Router &obj)
+{
+	if (this == &obj)
+		return *this;
+	_serverBlock = obj._serverBlock;
+	_FDsRef = obj._FDsRef;
+	_pollFd = obj._pollFd;
+	return *this;
+}
+
```

Comment:
Shoudl we "NULL" the values in the constructors?
Is the copy assignment operator good like this?